README.md

```diff
@@ -291,14 +296,19 @@ Leet list status=<status> difficulty=<difficulty>
 
 ## 🚀 Usage
 
-This plugin is meant to be used within a **fresh** [Neovim] instance.
-Meaning that to launch [leetcode.nvim] you **have** to pass
-[`arg`](#arg) as the _first and **only**_ [Neovim] argument
+This plugin can be initiated in two ways:
+
+- To start [leetcode.nvim], simply pass [`arg`](#arg)
+  as the _first and **only**_ [Neovim] argument
 
 ```
 nvim leetcode.nvim
 ```
 
+- _**(Experimental)**_ Alternatively, you can use `:Leet` command to open [leetcode.nvim]
+  within your preferred dashboard plugin. The only requirement is that [Neovim]
+  must not have any listed buffers open.
```

Comment:
```suggestion
- _**(Experimental)**_ Alternatively, you can use `:Leet` command to open [leetcode.nvim].
```

Reply:
Ah wait nevermind. That's just with the nonstandalone plugin, my bad! Do you want me to write the docs for that? Or the English ones at least haha

Reply:
If you want you can create a `plugins` sections and describe `nonstandalone` plugin. For chinese i just use chat gpt xD

Reply:
LOLL nice

Will do!